core/proto/core.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto2";
+
+package core;
+
+option go_package = "proto";
+
+message Challenge {
+	optional int64 id = 1;
+	optional string type = 2;
+	optional string status = 6;
+	optional string validatedTime = 8; // time.RFC3339Nano
```

Comment:
Why are we using a formatted time rather than an int64 of nanos since the epoch, which would be more compact?